### PR TITLE
fix path for molecule binary in container

### PR DESCRIPTION
**Pull request by fabianvf**
#### Describe what this PR does and why we need it:
Makes it so the molecule binary is in the PATH in the molecule container

### CODE REVIEW

File: operator/build/molecule.Dockerfile [hunk 1]
```diff
@@ -3,9 +3,10 @@ FROM centos:7
 RUN mkdir -p /home/molecule
 
 ENV HOME=/home/molecule
-ENV PYTHONUSERBASE=${HOME}/.bin
-RUN mkdir -p ${PYTHONUSERBASE}
-ENV PATH="${HOME}/.bin:${PATH}"
+ENV PYTHONUSERBASE=${HOME} \
+    PATH="${HOME}/bin:${PATH}"
```
Comment (jmrodri) [inline operator/build/molecule.Dockerfile]: You sure this is right? When I did `pip install --user molecule==2.20.0.0a2` on my machine it put it in `$HOME/.local/bin`

Comment (fabianvf) [reply operator/build/molecule.Dockerfile]: Yeah, the overridden PYTHONUSERBASE makes it just $HOME/bin

Comment (fabianvf) [reply operator/build/molecule.Dockerfile]: Yeah, the overridden PYTHONUSERBASE makes it just $HOME/bin

**Pull request merged**